doc/content/api-reference/html.rst

```diff
@@ -827,6 +827,73 @@ select is no multi select, all other options get unselected automatically.
     |style             |(Dict) contains all styling attributes
 
 
+RadioGroup and RadioButton
+++++++++++++++++++++++++++
+
+.. code-block:: python
+
+    from lona.html import RadioGroup, RadioButton, Label
+
+    radio_group = RadioGroup(
+        Label('Option 1', RadioButton(value=1)),
+        Label('Option 2', RadioButton(value=2.0)),
+        Label('Option 3', RadioButton(value='3'), checked=True),
+    )
```

Comment:
Initializing a RadioGroup via the constructor is not implemented! Using it this way Labels and RadioButtons are not linked together with the `for=` attribute.
Using `radio_group.add_button()` works however.

Reply:
That is due to the weird implementation of radio buttons in HTML. Correlating which `Label` is linked to which `RadioButton` is tricky when you get a whole tree of both. That's why, in this example, the `RadioButton` objects are inside of the `Label` objects. That way they are linked together by the browser implicitly. In `add_button()` that's not a problem, because you are supposed to provide only one label and one input with each call.

The `RadioGroup.value` is initialized correctly in this example because `value` is a property that searches for a `RadioButton` that is checked on every call. It is not a static value.

Reply:
OK. Good point. Having the nodes in this way makes the result look funny. But `add_button()` works as expected. So no further action needed here.